tests/lmeval/run_lmeval.py

```
@@ -0,0 +1,60 @@
+def parse_args():
+    """Parse model and config JSON arguments passed via command line."""
+    if len(sys.argv) < 3:
+        msg = "Usage: python script.py '<model>' '<config_json>'"
+        raise ValueError(msg)
+
+    model = sys.argv[1]
+
+    try:
+        config = json.loads(sys.argv[2])
+    except json.JSONDecodeError as e:
+        raise ValueError(f"Invalid JSON config: {e}")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)